public/scss/_forms.scss

```diff
@@ -37,10 +37,10 @@ form.cra-form {
     max-width: 80%;
 
     .w-3-4 {
-      width: 50%;
+      width: 338px;
     }
     .w-1-2 {
-      width: 30%;
+      width: 225px;
```

Comment:
Ah, ha. So we do want to cap the width but doing this changes the intent of the classname from a relative sized-element to an absolute one.

If we set both `width` and `max-width` properties, we can allow the field to keep its relative sizing but also cap its length at some rough ~number of character size.

Here's an example: https://jsbin.com/geqohusomi/edit?html,output

In the example, the width of the input is 50% unless it exceeds 30 characters (type in 30 characters to check), at which point it is capped.

We pretty much have 3 kinds of text input fields at this point:
- login code / SIN (sized around 30 characters)
- date of birth: very small inputs
- amounts of money: probably max out around 15 characters (although that doesn't include the dollar sign)

The existing app is a pretty good reference for these sizes (obviously they are bigger but we don't want to make the inputs too short), so play around setting 'ch' values as `max-widths` so that the inputs look pretty similar to currently but can still grow and shrink.

Almost there!

t